presto-product-tests/bin/run_on_docker.sh

```diff
@@ -35,6 +39,18 @@ function stop_unnecessary_hadoop_services() {
   docker exec ${HADOOP_MASTER_CONTAINER} supervisorctl stop zookeeper
 }
 
+function start_mapr_httpfs_service() {
+  HADOOP_MASTER_CONTAINER=$(hadoop_master_container)
+  docker exec ${HADOOP_MASTER_CONTAINER} chown -R mapr /opt/mapr/httpfs
+  docker exec ${HADOOP_MASTER_CONTAINER} maprcli node services -name httpfs -action start -nodes ${HADOOP_HOST_NAME}
+}
+
+function configure_maprclient() {
+  HADOOP_MASTER_CONTAINER=$(hadoop_master_container)
+  PRESTO_MASTER_CONTAINER=$(presto_master_container)
+  docker exec ${PRESTO_MASTER_CONTAINER} /opt/mapr/server/configure.sh -N ${HADOOP_HOST_NAME} -c -C ${HADOOP_HOST_NAME}:7222 -HS ${HADOOP_HOST_NAME}
```

Comment:
Is this port number 7222 configurable in the MapR container? this should be parameterized or atleast a constant.